linux-tkg-config/prepare

```diff
@@ -361,11 +361,11 @@ _tkg_srcprep() {
   if [ "$_compiler_name" != "-llvm" ]; then
     msg2 "Applying graysky's cpu opts patch"
     if [ "${_distro}" = "Arch" ]; then
-      tkgpatch="$srcdir/enable_additional_cpu_optimizations_for_gcc_v10.1%2B_kernel_v${opt_ver}.patch" && _tkg_patcher
+      tkgpatch="$srcdir/enable_additional_cpu_optimizations_for_gcc_v10.1+_kernel_v${opt_ver}.patch" && _tkg_patcher
```

Comment:
You should simply revert this line and it's all good! Archlinux doesn't follow what's in `install.sh`

Reply:
But what is this line than for in the first place? From what I gathered "Arch" is also used as default / fallback. The "%2B" escape does not work with `_tkg_patcher`, why should it be reverted?

Reply:
This line is used when the `prepare` script is called from the `PKGUILD` script. And the `PKGBUILD` script downloads the files with the exact same file name involving the `%2B`

Reply:
Thank You for explaining! I think I understand better now how it works for Arch. I reverted the line for Arch.